packages/cli/README.md

```diff
@@ -2,16 +2,28 @@
 
 CLI for interacting with https://MermaidChart.com, the platform that makes collaborating with Mermaid diagrams easy!
 
+## Installation
+
+`@mermaidcart/cli` is tested to work with Node.JS v18.18.0 or later.
+
+We recommend installation using [npx](https://docs.npmjs.com/cli/v10/commands/npx)
+to automatically download, cache, and run `@mermaidcart/cli`, as it comes with
+most Node.JS installations.
+
+```bash
+npx @mermaidchart/cli --help
+```
```

Comment:
We can't test this until we've published this package to NPM, but it should work!